test/integration/test-amp-bind.js

```diff
@@ -39,25 +39,28 @@ describe.configure().skipEdge().run('amp-bind', function() {
     /* eslint-enable max-len */
     extensions: ['amp-bind'],
   }, env => {
-    let doc, text;
+    let browser, doc, text;
+
+    this.timeout(TIMEOUT);
 
     beforeEach(() => {
       doc = env.win.document;
       text = doc.querySelector('p');
+      browser = new BrowserController(env.win);
     });
 
-    it('[text]', () => {
+    it('[text]', function*() {
```

Comment:
since we can do async/await in tests now, can do:

```javascript
it('[text]', async() => {
  await browser.wait(200);
});
```

Reply:
Let's keep consistency with the rest of the file in this diff. We can submit a separate PR to refactor the entire file.